### PR TITLE
Adjust roles for deploying application controller

### PR DESCRIPTION
**Pull request by barney-s**
- adjust roles since Application  needs to list and add ownerref to arbitrary resources
- add make undeploy target.

ps: we still need to change the image 

diff --git a/config/default/manager_image_patch.yaml b/config/default/manager_image_patch.yaml                                                                                                                                                
index fcbf39dc..3c7f3e27 100644                                                                                                                                                                                                               
--- a/config/default/manager_image_patch.yaml                                                                                                                                                                                                 
+++ b/config/default/manager_image_patch.yaml                                                                                                                                                                                                 
@@ -8,5 +8,5 @@ spec:                                                                                                                                                                                                                         
     spec:                                                                                                                                                                                                                                    
       containers:                                                                                                                                                                                                                            
       # Change the value of image field below to your controller image URL                                                                                                                                                                   
-      - image: IMAGE_URL                                                                                                                                                                                                                     
+      - image: << My image here >>
         name: manager

### CODE REVIEW

**Comment by barney-s** [comment]
fixes #99

File: config/rbac/rbac_role.yaml [hunk 1]
```diff
@@ -4,6 +4,12 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
+  - '*'
+  resources:
+  - '*'
+  verbs:
```
Comment (ant31) [inline config/rbac/rbac_role.yaml]: probably only "get, list, update, patch, watch"

Comment (barney-s) [reply config/rbac/rbac_role.yaml]: ack

**Pull request merged**